### ISSUE TITLE
Build failure on Jenkins

### ISSUE THREAD

**Issue by caldav** [ubuntudesign/snapcraft.io#212]
The Jenkins build of the site fails:
http://jenkins.demo.haus/job/snapcraft.io-code-build/92/console

Last successful build was on Oct. 3.

```
[...]
## Getting node dependencies

## Build CSS up

module.js:457
    throw err;
    ^

Error: Cannot find module './src/init'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/var/lib/jenkins/jobs/snapcraft.io-code-build/workspace/snapcraft.io/node_modules/gulp-sourcemaps/index.js:4:9)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
Build step 'Execute shell' marked build as failure
Finished: FAILURE
```

**Comment by barrymcgee** [comment]
@nottrobin Need your help here please - this seems to be the same error as outlined: http://stackoverflow.com/questions/40030923/gulp-sourcemaps-cannot-find-module-src-init

..which by following the trail, it would seem the fix is to is bump `gulp-sourcemaps` to `1.8.1` but that doesn't seem to have made any difference?

**Comment by barrymcgee** [comment]
Now building again - thanks @nottrobin 

http://jenkins.demo.haus/job/snapcraft.io-code-build/97/console
